2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_setup.py:_flush():70] setting env: {}
2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_init.py:_log_setup():319] Logging user logs to /home/naddeok5/FIM/wandb/run-20210216_153243-10zbgg68/logs/debug.log
2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_init.py:_log_setup():320] Logging internal logs to /home/naddeok5/FIM/wandb/run-20210216_153243-10zbgg68/logs/debug-internal.log
2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_init.py:init():324] calling init triggers
2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_init.py:init():331] wandb.init called with sweep_config: {}
config: {'epochs': 1, 'batch_size': 128, 'momentum': 0.9, 'weight_decay': 0.005, 'learning_rate': 0.01, 'optimizer': 'sgd', 'criterion': 'mse'}
2021-02-16 15:32:43,967 INFO    MainThread:28315 [wandb_init.py:init():401] starting backend
2021-02-16 15:32:43,991 INFO    MainThread:28315 [backend.py:ensure_launched():81] starting backend process...
2021-02-16 15:32:44,007 INFO    MainThread:28315 [backend.py:ensure_launched():86] started backend process with pid: 28743
2021-02-16 15:32:44,008 INFO    MainThread:28315 [wandb_init.py:init():410] backend started and connected
2021-02-16 15:32:44,011 INFO    MainThread:28315 [wandb_init.py:init():433] updated telemetry
2021-02-16 15:32:44,012 INFO    MainThread:28315 [wandb_init.py:init():456] communicating current version
2021-02-16 15:32:44,790 INFO    MainThread:28315 [wandb_init.py:init():461] got version response upgrade_message: "wandb version 0.10.19 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-02-16 15:32:44,790 INFO    MainThread:28315 [wandb_init.py:init():469] communicating run to backend with 30 second timeout
2021-02-16 15:32:44,936 INFO    MainThread:28315 [wandb_init.py:init():492] starting run threads in backend
2021-02-16 15:32:46,193 INFO    MainThread:28315 [wandb_run.py:_console_start():1411] atexit reg
2021-02-16 15:32:46,194 INFO    MainThread:28315 [wandb_run.py:_redirect():1274] redirect: SettingsConsole.REDIRECT
2021-02-16 15:32:46,195 INFO    MainThread:28315 [wandb_run.py:_redirect():1279] Redirecting console.
2021-02-16 15:32:46,195 INFO    MainThread:28315 [redirect.py:install():213] install start
2021-02-16 15:32:46,196 INFO    MainThread:28315 [redirect.py:install():228] install stop
2021-02-16 15:32:46,196 INFO    MainThread:28315 [redirect.py:install():213] install start
2021-02-16 15:32:46,196 INFO    MainThread:28315 [redirect.py:install():228] install stop
2021-02-16 15:32:46,196 INFO    MainThread:28315 [wandb_run.py:_redirect():1325] Redirects installed.
2021-02-16 15:32:46,196 INFO    MainThread:28315 [wandb_init.py:init():515] run started, returning control to user process
2021-02-16 15:32:54,322 INFO    MainThread:28315 [wandb_run.py:_atexit_cleanup():1381] got exitcode: 1
2021-02-16 15:32:54,323 INFO    MainThread:28315 [wandb_run.py:_restore():1353] restore
2021-02-16 15:32:54,323 INFO    MainThread:28315 [redirect.py:uninstall():232] uninstall start
2021-02-16 15:32:54,324 INFO    MainThread:28315 [redirect.py:_stop():287] _stop: stdout
2021-02-16 15:32:54,325 INFO    MainThread:28315 [redirect.py:_stop():293] _stop closed: stdout
2021-02-16 15:32:54,325 INFO    stdout    :28315 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-02-16 15:32:54,326 INFO    stdout    :28315 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-02-16 15:32:54,326 INFO    MainThread:28315 [redirect.py:_stop():299] _stop joined: stdout
2021-02-16 15:32:54,326 INFO    MainThread:28315 [redirect.py:_stop():301] _stop rd closed: stdout
2021-02-16 15:32:54,326 INFO    MainThread:28315 [redirect.py:uninstall():236] uninstall done
2021-02-16 15:32:54,326 INFO    MainThread:28315 [redirect.py:uninstall():232] uninstall start
2021-02-16 15:32:54,327 INFO    MainThread:28315 [redirect.py:_stop():287] _stop: stderr
2021-02-16 15:32:54,327 INFO    MainThread:28315 [redirect.py:_stop():293] _stop closed: stderr
2021-02-16 15:32:54,327 INFO    stderr    :28315 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-02-16 15:32:54,327 INFO    stderr    :28315 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-02-16 15:32:54,327 INFO    MainThread:28315 [redirect.py:_stop():299] _stop joined: stderr
2021-02-16 15:32:54,327 INFO    MainThread:28315 [redirect.py:_stop():301] _stop rd closed: stderr
2021-02-16 15:32:54,327 INFO    MainThread:28315 [redirect.py:uninstall():236] uninstall done
2021-02-16 15:32:54,456 INFO    MainThread:28315 [wandb_run.py:_wait_for_finish():1504] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 720
  total_bytes: 720
}

2021-02-16 15:32:56,462 INFO    MainThread:28315 [wandb_run.py:_wait_for_finish():1504] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3956
  total_bytes: 3956
}

2021-02-16 15:32:57,817 INFO    MainThread:28315 [wandb_run.py:_show_files():1726] logging synced files
